coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/CalciteTrinoUDFOperatorTransformers.java

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableList;
+
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+
+import com.linkedin.coral.com.google.common.base.CaseFormat;
+import com.linkedin.coral.com.google.common.base.Converter;
+import com.linkedin.coral.com.google.common.collect.ImmutableMultimap;
+import com.linkedin.coral.common.functions.Function;
+import com.linkedin.coral.common.transformers.SqlCallTransformer;
+import com.linkedin.coral.common.transformers.SqlCallTransformers;
+import com.linkedin.coral.common.transformers.StandardUDFOperatorTransformer;
+import com.linkedin.coral.hive.hive2rel.functions.HiveRLikeOperator;
+import com.linkedin.coral.hive.hive2rel.functions.StaticHiveFunctionRegistry;
+import com.linkedin.coral.trino.rel2trino.functions.TrinoElementAtFunction;
+import com.linkedin.coral.trino.rel2trino.transfomers.MapStructAccessOperatorTransformer;
+
+
+/**
+ * This class defines a list of SqlCallTransformers which convert UDF operator from Calcite to Trino
+ * on SqlNode layer
+ */
+public class CalciteTrinoUDFOperatorTransformers extends SqlCallTransformers {
```

Comment:
Do we need to extend this class? I do not think this class is meant for extension. Probably we can even call it `final`. @ljfgem thoughts?

Reply:
Agree that we don't extend this class.
`SqlCallTransformers` might be a static parameter of this class.
And the name is not accurate, what about `CoralToTrinoSqlCallTransformers`?

Reply:
Done.